Snakefile

```diff
@@ -153,21 +155,113 @@ rule iq_tree:
         rooted="output/strains/{strain}/besttree.nwk",
         unrooted=temp("output/strains/{strain}/besttree.unrooted.treefile"),
         iqtree=temp("output/strains/{strain}/besttree.unrooted.iqtree"),
-        ckp=temp("output/strains/{strain}/besttree.unrooted.ckp.gz")
+        ckp=temp("output/strains/{strain}/besttree.unrooted.ckp.gz"),
     log:
         "output/strains/{strain}/besttree.unrooted.log"
     params:
         enabled=config['iqtree']['enabled'],
         mode=config['iqtree']['mode'],
         model=config['iqtree']['model'],
-        prefix="output/strains/{strain}/besttree.unrooted"
+        prefix="output/strains/{strain}/besttree.unrooted",
+        temp="output/strains/{strain}/temp_align_variants.aln"
+
     conda:
         config["poppipe_location"] + "/envs/iqtree.yml"
     threads:
-        16
+        4
     script:
         config["poppipe_location"] + "/scripts/run_iqtree.py"
 
+rule gubbins:
+    input:
+        alignment="output/strains/{strain}/align_variants.aln",
+        start_tree="output/strains/{strain}/besttree.nwk"
+    output:
+        final_tree="output/strains/{strain}/gubbins.final_tree.tre",
+    group:
+        "gubbins"
+    # log:
+    #     "logs/gubbins_{strain}.log"
+    params:
+        prefix=config['gubbins']['prefix'],
+        tree_builder=config['gubbins']['tree-builder'],
+        min_snp=config['gubbins']['min-snps'],
+        min_window=config['gubbins']['min-window-size'],
+        max_window=config['gubbins']['max-window-size'],
+        iterations=config['gubbins']['iterations'],
+        directory=config['gubbins']['dir'],
```

Comment:
This should just be set, not read from the config (see other comment on config.yml)